### DISCUSSION TITLE
[HELP] Error - Storage Error Notification

### DISCUSSION THREAD

**Discussion by CosmasSC128** [Budibase/budibase#10559] [Help]
Recently got this error for the first time I've never seen before. It is attached as well.

"Storage backend has reached its minimum free drive threshold. Please delete a few objects to proceed."

This is while trying to use the budibase upload widget.

Appreciate any help, thank you.

<img width="949" alt="MicrosoftTeams-image" src="https://github.com/Budibase/budibase/assets/93920520/be41ea6c-fd59-40ca-8394-aa2ab274641f">

**Comment by ConorWebb96** [comment]
Hey @CosmasSC128,

Are you self-hosted or cloud based?

If you are self-hosted can you provide a bit of information on what you are user for it? (k8s, docker single image, docker compose, digital ocean)

Can you also let us know what version of Budibase you are using.

**Reply by CosmasSC128** [reply]
Sorry, correction (it's different from the past now), it is on azure kubernetes, installed using helm.

**Reply by ConorWebb96** [reply]
Hey @CosmasSC128,

Since you are using the Helm you should be able to update that storage property for example to `200Mi` and update the deployment. This would normally be done with something like `helm update -f values.yaml budibase/budibase` however this many vary depending on your setup.

Hopefully this points you in the right direction!

**Reply by PClmnt** [reply]
@CosmasSC128 

Before doing the above we'd actually like to investigate a bit further, could you send an email to `support@budibase.com` with the following

- Your k8s values.yaml file 
- The output of `kubectl get pvc minio-data -o yaml`

You may also want to backup your system in the case of making any configuration changes

**Reply by CosmasSC128** [reply]
I have emailed them.
